lib/matplotlib/patheffects.py

```diff
@@ -373,3 +375,168 @@ def draw_path(self, renderer, gc, tpath, affine, rgbFace):
         if clip_path:
             self.patch.set_clip_path(*clip_path)
         self.patch.draw(renderer)
+
+
+class TickedStroke(AbstractPathEffect):
+    """
+    A line-based PathEffect which draws a path with a ticked style.
+    This line style is frequently used to represent constraints in
+    optimization.  The ticks indicate that one side of the line is
+    invalid or to represent a closed boundary of a domain (i.e. a
+    wall or the edge of a pipe).
+
+    The spacing, length, and angle of ticks can be controlled.
+
+    This line style is sometimes referred to as a hatched line.
+
+    See also the :doc:`contour demo example
+    </gallery/lines_bars_and_markers/lines_with_ticks_demo>`.
+
+    See also the :doc:`contours in optimization example
+    </gallery/images_contours_and_fields/contours_in_optimization_demo>`.
+    """
+
+    def __init__(self, offset=(0, 0),
+                 spacing=10.0, angle=45.0, length=np.sqrt(2),
+                 flip=False, **kwargs):
+        """
+        Parameters
+        ----------
+        offset : pair of floats
+            The offset to apply to the path, in points.
+        spacing : float
```

Comment:
Default values belong on this line, not the end of the parameter description. https://matplotlib.org/devel/documenting_mpl.html#default-values

Reply:
Done